cypress/support/fragments/inventory/instanceRecordEdit.js

```diff
@@ -180,4 +182,13 @@ export default {
       cy.expect(Checkbox({ name: 'previouslyHeld' }).has({ checked: true }));
     } else cy.expect(Checkbox({ name: 'previouslyHeld' }).has({ checked: false }));
   },
+  editResourceTitle: (newTitle) => {
+    cy.do(TextArea({ name: 'title' }).fillIn(newTitle));
+    cy.expect(TextArea({ name: 'title' }).has({ value: newTitle }));
+  },
+  verifySuccessfulMessage: () => {
```

Comment:
I recommend not to introduce new function but add verification to saveAndClose method:
```JS
    saveAndClose({ instanceSaved = false } = {}) {
    cy.wait(1500);
    cy.do(saveAndCloseButton.click());
    cy.expect([actionsButton.exists(), PaneHeader(including('Edit instance')).absent()]);

    if (instanceSaved) {
      InteractorsTools.checkCalloutMessage(
        matching(new RegExp(InstanceStates.instanceSavedSuccessfully)),
      );
    }
  },
```

and call 'saveAndClose' method in test with the following params:
```JS
saveAndClose({ instanceSaved = true });
```